fast_pauli/cpp/include/__summed_pauli_op.hpp

```diff
@@ -613,6 +613,27 @@ template <std::floating_point T> struct SummedPauliOp
         }
     }
 
+    /**
+     * @brief Return the components of the SummedPauliOp as a vector of PauliOps
+     *
+     * @return std::vector<PauliOp<T>>
```

Comment:
Is this primarily for debugging?

Reply:
Not only that, it's also used in some places where we need to use PauliOp directly. For example, when we need to multiply specific PauliOp with PauliString